Lib/traceback.py

```diff
@@ -328,13 +332,27 @@ def _original_line(self):
         self.line
         return self._line
 
+    @property
+    def _dedented_lines(self):
+        # Returns _original_line, but dedented
+        self.line
```

Comment:
Empty statement?

Reply:
This is intended to call the `line` property, which sets `self._line`.

Reply:
Ugh  I find this is confusing and it may be pointing out this should not be a property. @lysnikolaou wdyt?

Reply:
Refactored some of the `line` property into a separate function.